setup.py

```diff
@@ -65,7 +65,8 @@
                       'sortedcontainers==1.5.7', 'psutil', 'pip',
                       'intervaltree'],
     extras_require={
-        'stats': ['python-firebase']
+        'stats': ['python-firebase'],
+        'benchmark': ['pympler']
```

Comment:
Just in case: 
Should we take care of about licence of `pympler` and `python-firebase`?
Do they have friendly licence?